### DISCUSSION TITLE
NeoGeo and its bios.

### DISCUSSION THREAD

**Discussion by Nikoh77** [emmercm/igir#857] [Q&A]
Talking about NeoGeo and using the official libretro neogeo only dat files on this page: https://github.com/libretro/FBNeo/tree/master/dats
buiding a romset with --merge-roms split each parent contains all the bios files inside it.
That's right?
Instead I would like the bios to be only in the external neogeo.zip file, is it possible?
Also, if I wanted to separate generic arcade games from those of neogeo, is there a way to filter them from the general arcade dat file?
Thanks

**Comment by Nikoh77** [comment]
This from FinalBurn Neo discord server... basically they tell me that most likely there is a bug in IGIR because clrmamepro behaves differently:
![Schermata a 2023-12-27 17-45-55](https://github.com/emmercm/igir/assets/7826178/27e9e963-8796-41a8-82fd-be5e0334b4bb)
![Schermata a 2023-12-27 17-46-38](https://github.com/emmercm/igir/assets/7826178/c1c5d272-6776-4c9b-86bf-0a5df17dc296)

**Reply by emmercm** [reply]
I've mostly tested with MAME DATs, let me do some testing with FBNeo.

**Reply by emmercm** [reply]
@Nikoh77 could you see if this fixes your issue, it has instructions in a comment: https://github.com/emmercm/igir/pull/880
